pkg/status/status.go

```diff
@@ -200,3 +227,71 @@ func (s *status) updateStatus() {
 		log.Fatalf("Error %v updating ClusterOperator", err)
 	}
 }
+
+// syncChannelConsumer will listen on the sync channel and update the status'
+// syncs and successfulSyncs fields until the stopCh is closed.
+func syncChannelConsumer(instance *status, stopCh <-chan struct{}) {
+	log.Info("Starting sync consumer")
+	go wait.Until(func() {
+		// Listen on SyncCh and update instance accordingly
+		err := <-syncCh
+		instance.syncs++
+		if err == nil {
+			instance.successfulSyncs++
+		}
+		log.Infof("Successful Syncs / Total Syncs: %d/%d", instance.successfulSyncs, instance.syncs)
```

Comment:
Will this make the log verbose?

Reply:
maybe we should convert this to `debug`?